src/main/java/com/teamscale/upload/utils/FileSystemUtils.java

```diff
@@ -327,7 +327,7 @@ public static void ensureEmptyFile(File file) throws IOException {
 		if (file.isDirectory()) {
 			throw new IOException("Unable to create empty file because it is a directory: " + file);
 		}
-		if (file.exists() && !file.delete()) {
+		if (file.exists()) {
```

Comment:
TODO: Check whether this method can be removed or think again whether this changes makes sense